src/index.js

```diff
@@ -4,8 +4,6 @@ import { enableLiveReload } from 'electron-compile'
 require('./renderer-events')
 
 let mainWindow
-const isDevMode = process.execPath.match(/[\\/]electron/)
-if (isDevMode) enableLiveReload({ strategy: 'react-hmr' })
```

Comment:
The linter is failing because we are now importing but not using `enableLiveReload` in this file.